lib/Parser.js

```diff
@@ -580,24 +580,26 @@ Parser.prototype.walkExportNamedDeclaration = function walkExportNamedDeclaratio
 				var pos = this.scope.definitions.length;
 				this.walkStatement(statement.declaration);
 				var newDefs = this.scope.definitions.slice(pos);
-				newDefs.reverse().forEach(function(def, idx) {
-					this.applyPluginsBailResult("export specifier", statement, def, def, idx);
-				}, this);
+				for(var index = newDefs.length - 1; index >= 0; index--) {
+					var def = newDefs[index];
+					this.applyPluginsBailResult("export specifier", statement, def, def, index);
+				}
 			}
 		}
 	}
 	if(statement.specifiers) {
-		statement.specifiers.forEach(function(specifier, idx) {
+		for(var specifierIndex = 0; specifierIndex <= statement.specifiers.length - 1; specifierIndex++) {
```

Comment:
Shouldn't you assign `statement.specifiers.length - 1` to a variable, like
```
for(var specifierIndex = 0, len = statement.specifiers.length - 1; specifierIndex <= len; specifierIndex++)
```
Otherwise, it'll evaluate every iteration.